second session/assignment/shoppingmall.js

```diff
@@ -0,0 +1,123 @@
+const COPYEDITEMS = [
+    { 
+        category : "운동용품",
+      imgSrc: "./static/img/자전거.jpg",
+      altText: "자전거",
+      name: "자전거",
+      price: "10000"
+    },
```

Comment:
p5) 상수를 따로 분리해서 js 파일을 만들고, export 키워드를 사용해 다른 모듈에서 불러와서 쓸 수 있게해도 좋겠네요! 
관련 링크 첨부해드립니다!
https://ko.javascript.info/import-export

```suggestion
export const COPYEDITEMS = [
    { 
        category : "운동용품",
      imgSrc: "./static/img/자전거.jpg",
      altText: "자전거",
      name: "자전거",
      price: "10000"
    },
```

Reply:
저도 처음에 export를 사용하려고 했는데 실패했습니다.. 한번다시 해보겠습니다.